lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -8,18 +8,30 @@ def self.run(params)
         require_relative '../../helper/android/android_version_helper'
         require_relative '../../helper/android/android_git_helper'
 
+        build_gradle_path = params[:build_gradle_path]
+        version_properties_path = params[:version_properties_path]
+
         # Checkout default branch and update
         default_branch = params[:default_branch]
         Fastlane::Helper::GitHelper.checkout_and_pull(default_branch)
 
         # Check versions
-        release_version = Fastlane::Helper::Android::VersionHelper.get_release_version
+        release_version = Fastlane::Helper::Android::VersionHelper.get_release_version(
```

Comment:
Can we use named parameters here? Makes it much easier to do backcompat (for instance, if we want to re-order the parameters), and easier to follow along in general.

Same for basically every instance of passing these objects around :)

Reply:
Added named parameters here: https://github.com/wordpress-mobile/release-toolkit/commit/73b6537b2dd20f5fb7f8bf6f3c126febb5d48cfb